modules/navier_stokes/test/tests/finite_volume/limiters/lid-driven/test.i

```diff
@@ -128,13 +131,20 @@ rho=1
 []
 
 [Executioner]
-  type = Steady
+  type = Transient
```

Comment:
Because of the way in which we limit using the explicit limiting flux computed from the previous step. Steady solves won't be really getting the right limiting

Reply:
Add warning in the documentation